File: PRO/codigos2324/SESIONES EJERCICIOS/EjerciciosExamen/src/main/java/ejercicio3/Usuario.java
package ejercicio3;

public class Usuario {

    private String nombre, correo, pass;

    public Usuario() {
    }

    public Usuario(String nombre, String correo, String pass) {
        this.nombre = nombre;
        this.correo = correo;
        this.pass = pass;
    }

    public void mostrarDatos(){
        System.out.println("nombre = " + nombre);
        System.out.println("correo = " + correo);
        System.out.println("pass = " + pass);
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getCorreo() {
        return correo;
    }

    public void setCorreo(String correo) {
        this.correo = correo;
    }

    public String getPass() {
        return pass;
    }

    public void setPass(String pass) {
        this.pass = pass;
    }
}

File: DWEC/curso2526/T3/logica/utils/js/usuario.js
class Usuario {
  nombre;
  apellido;
  correo;
  fecha;

  //public Usuario(){}
  constructor(nombre, apellido, correo, fecha) {
    this.nombre = nombre;
    this.apellido = apellido;
    this.correo = correo;
    this.fecha = fecha;
  }

  // public String getNombre() {return nombre}
  // getter
  getNombre() {
    return this.nombre;
  }
  // setter
  setNombre(nombre) {
    this.nombre = nombre;
  }

  getCorreo() {
    return this.correo;
  }

  setCorreo(correo) {
    this.correo = correo;
  }

  mostrarDatos() {
    return `${this.nombre} ${this.apellido}`;
  }

  getDominio() {
    return this.correo.substring(this.correo.indexOf("@"));
  }
}
